sql/src/main/java/io/crate/analyze/expressions/ExpressionAnalyzer.java

```diff
@@ -853,6 +878,38 @@ protected Symbol visitSubqueryExpression(SubqueryExpression node, ExpressionAnal
 
     }
 
+
+    private Symbol allocateFunction(FunctionInfo functionInfo,
+                                    List<Symbol> arguments,
+                                    ExpressionAnalysisContext context) {
+        return allocateFunction(functionInfo, arguments, context, functions);
+    }
+
+    /**
+     * Creates a function symbol and tries to normalize the new function's
+     * {@link FunctionImplementation} iff only Literals are supplied as arguments.
```

Comment:
`iff` typo

Reply:
`iff` typically stands for [`if and only if`](https://en.wikipedia.org/wiki/If_and_only_if)